.github/workflows/docker-publish.yml

```
@@ -13,7 +13,7 @@ jobs:
       - id: build_info
         name: build info
         run:
-          echo ::set-output name=tags::ghcr.io/${GITHUB_REPOSITORY//docker-/}:${GITHUB_REF#refs/tags/},ghcr.io/${GITHUB_REPOSITORY//docker-/}:latest
+          echo ::set-output name=tags::ghcr.io/${GITHUB_REPOSITORY//docker-/}:v-${GITHUB_REF#refs/tags/},ghcr.io/${GITHUB_REPOSITORY//docker-/}:latest
```

Moi je n'aime pas beaucoup les tags v-1.0.110, pourquoi pas v1.0.110?
```suggestion
          echo ::set-output name=tags::ghcr.io/${GITHUB_REPOSITORY//docker-/}:v${GITHUB_REF#refs/tags/},ghcr.io/${GITHUB_REPOSITORY//docker-/}:latest
```